packages/world/src/utils/OreLib.sol

```diff
@@ -47,8 +37,12 @@ library OreLib {
   }
 
   function burnOre(ObjectType self, uint256 amount) internal {
+    uint256 resourceCount = ResourceCount._get(self);
+
+    require(resourceCount >= amount, "Not enough resources to burn");
```

Comment:
ahh nice, this is a better error message than the arithmetic solidity error lol

Reply:
yeah, got tired of debugging the same issue over and over when tests broke lol